src/api/paraswap/paraswap.spec.ts

```diff
@@ -131,7 +130,7 @@ describe('Get Quote', () => {
     // WHEN
     expect.assertions(1)
     // user forces TS to accept chain as valid
-    const promise = cowSdk.paraswapApi!.getQuote({ ...query, chainId: 2 as NetworkID })
+    const promise = cowSdk.paraswapApi.getQuote({ ...query, chainId: 2 as NetworkID })
```

Comment:
is the `NetworkID ` cast necessary?

Reply:
No, it was supposed to represent a user of the SDK bypassing TS errors and forcing 2 as an accepted type parameter. But it isn't strictly required in the test

Reply:
now I see, I lacked context, what about using [@ts-expect-error](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments)?

Reply:
good idea